mscore/musescore.h

```diff
@@ -366,7 +366,7 @@ class MuseScore : public QMainWindow, public MuseScoreCore {
 
       bool _midiinEnabled            { true };
       QList<QString> plugins;
-      ScriptEngine* se               { 0 };
+      ScriptEngine* scriptEngine     { 0 };
```

Comment:
Maybe we can even get rid of this entirrely, it doesn't seem to get used anywhere, @dmitrio95 ?

Reply:
OK, fine by me